packages/indexer-agent/src/commands/start.ts

```diff
@@ -194,10 +194,22 @@ export default {
         default: true,
         group: 'Indexer Infrastructure',
       })
-      .option('allocation-claim-threshold', {
-        description: `Minimum query fees (in GRT) received for an allocation to collect and claim them on-chain`,
-        type: 'number',
-        default: 0,
+      .option('rebate-claim-min-single-value', {
+        description: `Minimum value of query fees for a single allocation (in GRT) in order for it to be included in a batch rebate claim on-chain`,
+        type: 'string',
+        default: '0',
+        group: 'Indexer Infrastructure',
+      })
+      .option('rebate-claim-min-batch-value', {
+        description: `Minimum total value of query fees in an allocation batch (in GRT) before rebates are claimed on-chain`,
+        type: 'string',
+        default: '0',
+        group: 'Indexer Infrastructure',
+      })
+      .option('voucher-redeem-min-single-value', {
```

Comment:
Do you think we can simplify the configs by combining 'voucher-redeem-min-single-value' and 'rebate-claim-min-single-value'?  That is what we currently do, but I suppose some indexers may want to redeem vouchers even though they won't necessarily claim the rebate.